bedrock/firefox/templates/firefox/nothing-personal/index.html

```diff
@@ -0,0 +1,149 @@
+
+            <h5>Ok, what’s the privacy-catch?</h5>
+            <p><a href="{{ url('firefox.privacy.index') }}"">We don’t keep track of where you go</a> because 1. Yawn, and 2. We really don’t need it.</p>
+          </div>
+          <div class="c-sticky-note c-attached-sticky">
+
+           <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+           <br>
+           <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+           <br>
+           <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+            <p>Ok, bye.</p>
+
+          </div>
+        </div>
+        {% endcall %}
+    </div>
+
+    <div class="c-sticky-note c-detached-sticky">
+      <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+      <br>
+      <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+      <br>
+      <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
+      <p>Ok, bye.</p>
+    </div>
+
+    <div class="c-sign-off">
+      <div class="mzp-l-content c-nothing-personal">
+        <p>Nothing personal. Just browsing.</p>
+        <div class="c-sign-off-wordmark">
+          <img src="{{ static('img/firefox/nothing-personal/firefox-by-mozilla-wordmark-white.svg') }}" alt="" width="180" height="40" class="c-header-logo">
+        </div>
+      </div>
+    </div>
+    </section>
+    <aside>
+      <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" aria-label="Link to a Youtube video" target="_blank" rel="external noopener"><img src="{{ static('img/firefox/nothing-personal/folder-icon.svg') }}" alt="">Folder <br> <.< </a>
+      <a href="https://open.spotify.com/playlist/0vvXsWCC9xrXsKd4FyS8kM" aria-label="Link to a Spotify playlist" target="_blank" rel="external noopener"><img src="{{ static('img/firefox/nothing-personal/music-icon.svg') }}" alt="">focus <br> playlist</a>
```

Comment:
**question:** I see you have `aria-label` in use here to make the links a bit more understandable, but would it also be worth adding `aria-hidden` on the link text? e.g. `Folder <br> <.< ` and `focus <br> playlist`

Reply:
Since the link text is within the anchor tag with the `aria-label` attribute, then the `aria-label` will take priority with screen readers instead of the link text:

> the aria-label text will override the text supplied within the link.
> - https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA8.html

I also tested it with the screenreader, and only the `aria-label` attribute is being read aloud